src/prettytable/prettytable.py

```diff
@@ -1801,6 +1830,8 @@ def _stringify_row(self, row, options, hrule):
             lines = value.split("\n")
             new_lines = []
             for line in lines:
+                if line == "None" and self.none_format[field] is not None:
```

Comment:
So if the cell contains either `None` or the string `"None"`, it'll be replaced (e.g. by "N/A").

Is this intentional and/or acceptable?

(I don't have a strong preference either way, just wanted to check.)

Reply:
Hi Hugo,

Thanks for the kind comments and for making this process less intimidating than I thought it would be.

Yes, that was the intended behavior. The idea was to allow the user to replace any "None" values with a value of their choosing. So in my tests I used "N/A" (not available) to represent a None value.

After some thought, I don't know if the way I did it is best. I am thinking there could be a possibility where, after `lines = value.split("\n")`, there could be a "None" value in the split text for some other reason.

Do you think a better evaluation would be to see if `lines` has only a single value of "None" and then replace "None" with the `none_format` value?

Reply:
I think the way you've done it is fine, let's also include it in a test case (I'll add a suggestion below).

Thanks!